src/components/Api.js

```diff
@@ -0,0 +1,168 @@
+class Api {
+  constructor(options) {
+    this._authorization = "fd5203a5-1f57-4428-b210-42e565daa250";
+  }
+
+  getUserInfo() {
+    return fetch("https://around-api.en.tripleten-services.com/v1/users/me", {
+      headers: {
+        authorization: this._authorization,
+      },
+    })
+      .then((res) => {
+        if (res.ok) {
+          return res.json();
+        }
+        return Promise.reject(`Error: ${res.status}`);
+      })
+      .then((res) => {
+        res.description = res.about;
+        return res;
+      })
+      .catch((err) => console.error(err));
```

Comment:
Needs correction: Errors should be handled outside of the api class in `index.js`. Now `catch` block is placed in the middle of the chain, so the rest of the chain won't know anything about the error — but it should because we need to update the data only in case of success